packages/manager/modules/pci/src/projects/project/kubernetes/details/service/terminate/terminate.html

```diff
@@ -9,14 +9,11 @@
                data-on-dismiss="$ctrl.goBack()"
                data-loading="$ctrl.isDeleting"
                data-type="warning">
-        <p class="oui-paragraph" data-translate="kube_service_terminate_warning"></p>
-        <oui-field data-label="{{:: 'kube_service_terminate_enter' | translate }}">
-            <input class="oui-input"
-                   type="text"
-                   name="kubeClusterTerminateConfirm"
-                   data-ng-model="$ctrl.terminateConfirmation"
-                   data-ng-pattern="$ctrl.TERMINATE_INPUT"
-                   required>
-        </oui-field>
+        <p class="oui-paragraph"
```

Comment:
```suggestion
        <p
```

I think `oui-paragraph` is applied anyway ,isn't it ?

Reply:
You are right. The class is now removed